features/networking/multicast.feature

```diff
@@ -212,16 +212,17 @@ Feature: testing multicast scenarios
     Then the step should succeed
     And the output should match:
       | eth0\s+1\s+239.255.254.24 |
-    And I wait up to 10 seconds for the steps to pass:
+    And I wait up to 20 seconds for the steps to pass:
     """
     When I execute on the "<%= cb.proj1pod3 %>" pod:
       | cat | /tmp/proj1pod3.log |
     Then the step should succeed
     And the output should match:
       | <%= cb.proj1pod1ip %>.*joined \(S,G\) = \(\*, 239.255.254.24\), pinging |
       | <%= cb.proj1pod2ip %>.*joined \(S,G\) = \(\*, 239.255.254.24\), pinging |
-      | <%= cb.proj1pod1ip %>.*multicast, xmt/rcv/%loss = 5/5/0%                |
-      | <%= cb.proj1pod2ip %>.*multicast, xmt/rcv/%loss = 5/5/0%                |
+    And the output should not contain:
```

Comment:
`.*` is used in below table. Perhaps you need to use `match`, instead of `contain`

Reply:
@liangxia Fixed to use match.